worlds/bumpstik/__init__.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2022 FelicitusNeko
+#
+# This software is released under the MIT License.
+# https://opensource.org/licenses/MIT
+
+from BaseClasses import Item, MultiWorld, Tutorial, ItemClassification
+from .Items import BumpStikItem, item_table, item_groups
+from .Locations import location_table
+from .Options import *
+from .Regions import create_regions
+from ..AutoWorld import World, WebWorld
```

Comment:
Change this to be an absolute import (so we have .apworld format support)

```suggestion
from worlds.AutoWorld import World, WebWorld
```